crates/ruff/src/rules/flake8_implicit_str_concat/rules.rs

```diff
@@ -19,6 +36,33 @@ impl Violation for SingleLineImplicitStringConcatenation {
 }
 
 define_violation!(
+    /// ## What it does
+    /// Checks if there are implicitly concatenated strings on multiple
+    /// lines.
+    ///
+    /// ## Why is this bad?
+    /// For long string literals that are wrapped across multiple lines, the
+    /// PEP-8 style guide recommends to perform string concatenation
+    /// implicitly within parentheses instead of using a backslash for line
+    /// continuation. The former code style is arguably more readable than
+    /// the latter, even though both are syntactically equivalent.
+    ///
+    /// ## Example
+    /// ```python
+    /// z = "The quick brown fox jumps over the lazy "\
+    ///     "dog."
+    /// ```
+    ///
+    /// Use instead:
+    /// ```python
+    /// z = (
+    ///     "The quick brown fox jumps over the lazy "
+    ///     "dog."
+    /// )
+    ///
+    /// ## References
+    /// * [PEP 8](https://peps.python.org/pep-0008/#maximum-line-length)
+    /// ```
```

Comment:
Code block brackets should be placed just after the example (now include references)

Reply:
Sorry Not quite sure I follow, can you illustrate what you mean?

Reply:
The three backticks (```) need to be moved up a couple of lines.

Reply:
oh yea, nice catch, thanks!